.github/workflows/basket-api-deploy.yml

```
@@ -0,0 +1,50 @@
+name: Deploy basket-api
+
+on:
+  workflow_dispatch:
+  repository_dispatch:
+    types:
+    - deploy
+  workflow_run:
+    workflows: ["basket-api"]
+    branches: [dev]
+    types: [completed]
+
+env:
+  chart: basket-api
+  namespace: eshop
+  app-name: eshop
+  chart-root: deploy/k8s/helm
```

```suggestion
env:
  BASKET_API_CHART: basket-api
  NAMESPACE: eshop
  APP_NAME eshop
  CHART_ROOT: deploy/k8s/helm
```

I'm curious why we're not imposing (what I believed to be) the industry standard for `env` variables, with regard to character casing and what not. This would need to flow through all the places where env vars are referenced. Another benefit is that this aligns with existing content for GitHub Actions with .NET in our docs, and aligns with what GitHub does out of the box - see their `secrets` context, and the naming conventions there.

---

@IEvangelist I'm not sure there is a recognized standard for naming - many companies impose their own standards. Consistency is more important than the actual form - but happy to match the other docs! Templates would make this much easier 😄